python/tk_multi_workfiles/work_area.py

```diff
@@ -205,23 +205,23 @@ def _load_settings(self):
         settings_to_find = ["saveas_default_name", "saveas_prefer_version_up", 
                             "version_compare_ignore_fields", "file_extensions"]
         resolved_settings = {}
+        app = sgtk.platform.current_bundle()
         try:
             if self._context:
                 resolved_settings = self._get_settings_for_context(self._context, templates_to_find, settings_to_find)
         except:
-            # (TODO) - propogate problems up - maybe add an is_valid() method?
-            pass
+            app.log_exception("There was an error parsing the settings for context '%s'" % self._context)
```

Comment:
This is to help a bit with error reporting in workfiles. Everytime we end up here it's because some typo was introduced inside an environment which prevented us from loading it. This will let the user know what broke. Ideally it would be in the UI, but the console will have to do for now.